crates/red_knot_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -352,6 +359,13 @@ impl<'db> UseDefMap<'db> {
         self.bindings_iterator(self.public_symbols[symbol].bindings())
     }
 
+    pub(crate) fn attribute_assignments(
```

Comment:
Just for naming consistency I suppose we could call this `instance_attribute_bindings`?